lighty-examples/lighty-community-aaa-restconf-app/src/main/assembly/resources/sampleConfigSingleNode.json

```diff
@@ -19,28 +19,15 @@
-                { "nameSpace":"urn:opendaylight:params:xml:ns:yang:controller:config:distributed-datastore-provider","name":"distributed-datastore-provider","revision":"2014-06-12"},{ "nameSpace":"urn:opendaylight:params:xml:ns:yang:controller:md:sal:cluster:admin","name":"cluster-admin","revision":"2015-10-13"},
-                { "nameSpace":"urn:ietf:params:xml:ns:yang:ietf-lisp-address-types","name":"ietf-lisp-address-types","revision":"2015-11-05"},
-                { "nameSpace":"urn:ietf:params:xml:ns:yang:ietf-restconf-monitoring","name":"ietf-restconf-monitoring","revision":"2017-01-26"},
-                { "nameSpace":"urn:ietf:params:xml:ns:yang:ietf-restconf","name":"ietf-restconf","revision":"2017-01-26"},
-                { "nameSpace":"urn:ietf:params:xml:ns:yang:ietf-yang-library","name":"ietf-yang-library","revision":"2019-01-04"},
-                { "nameSpace":"urn:opendaylight:params:xml:ns:yang:controller:md:sal:remote","name": "sal-remote","revision":"2014-01-14"},
-                { "nameSpace":"urn:sal:restconf:event:subscription","name": "sal-remote-augment","revision":"2014-07-08"},
-                { "nameSpace":"subscribe:to:notification","name": "subscribe-to-notification","revision":"2016-10-28"},
-                { "nameSpace":"urn:opendaylight:yang:aaa:cert:mdsal","name": "aaa-cert-mdsal","revision":"2016-03-21"},
-                { "nameSpace":"urn:opendaylight:params:xml:ns:yang:aaa","name": "aaa","revision":"2016-12-14"},
-                { "nameSpace":"config:aaa:authn:encrypt:service:config","name": "aaa-encrypt-service-config","revision":"2016-09-15"},
-                { "nameSpace":"urn:TBD:params:xml:ns:yang:network-topology","name":"network-topology","revision":"2013-10-21"}
+                {"nameSpace":"urn:sal:restconf:event:subscription","name":"sal-remote-augment","revision":"2014-07-08"},
+                {"nameSpace":"subscribe:to:notification","name":"subscribe-to-notification","revision":"2016-10-28"},
+                {"nameSpace":"urn:ietf:params:xml:ns:yang:ietf-restconf","name":"ietf-restconf","revision":"2017-01-26"},
+                {"nameSpace":"urn:opendaylight:params:xml:ns:yang:controller:md:sal:remote","name":"sal-remote","revision":"2014-01-14"},
+                {"nameSpace":"urn:ietf:params:xml:ns:yang:ietf-restconf-monitoring","name":"ietf-restconf-monitoring","revision":"2017-01-26"},
+                {"nameSpace":"urn:opendaylight:yang:aaa:cert:mdsal","name":"aaa-cert-mdsal","revision":"2016-03-21"},
+                {"nameSpace":"urn:opendaylight:params:xml:ns:yang:aaa","name":"aaa","revision":"2016-12-14"},
+                {"nameSpace":"config:aaa:authn:encrypt:service:config","name":"aaa-encrypt-service-config","revision":"2016-09-15"},
+                {"nameSpace":"urn:ietf:params:xml:ns:yang:ietf-yang-library","name":"ietf-yang-library","revision":"2019-01-04"}
```

Comment:
This config is not working with community-aaa-restconf-app example. Fix missing model when this json example is used.

ERROR [main] (Main.java:85) - Lighty.io, Restconf and AAA module main application exception: 
java.util.concurrent.ExecutionException: org.opendaylight.mdsal.binding.dom.codec.api.MissingSchemaException: Module QNameModule{ns=urn:opendaylight:params:xml:ns:yang:controller:config:distributed-datastore-provider, rev=2014-06-12} is not present in current schema context.